core/jazz_cloud-logs-streamer/_ES/applicationlogs.json

```diff
@@ -6,15 +6,15 @@
   "mappings" : {
     "applicationlogs" : {
        "properties" : {
-			"timestamp" : { "type":"date","format":"dateOptionalTime" },
-			"request_id" : { "type": "string", "index": "not_analyzed" },
-			"platform_log_group" : { "type":"string","index":"not_analyzed" },
-			"platform_log_stream" : { "type":"string","index":"not_analyzed" },
-			"servicename" : { "type": "string", "index": "not_analyzed" },
-			"domain" : { "type": "string", "index": "not_analyzed" },
-      "environment" : { "type": "string", "index": "not_analyzed" },
-      "log_level" : { "type": "string", "index": "not_analyzed" },
-			"message" : { "type": "string", "index": "not_analyzed" }
+                        "timestamp" : { "type":"date","format":"dateOptionalTime" },
+                        "request_id" : { "type": "keyword", "index": "false" },
```

Comment:
I think `index` should be `true` according to this [guide](https://www.elastic.co/blog/strings-are-dead-long-live-strings).

Reply:
Done